src/main/java/brandkon/Product/ProductController.java

```diff
@@ -0,0 +1,40 @@
+package brandkon.Product;
+
+import brandkon.Brand.Brand;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class ProductController {
+
+    private final ProductService productService;
+
+    public ProductController(ProductService productService) {
+        this.productService = productService;
+    }
+
+    // 상품 목록 조회
+    @GetMapping("/products")
+    public List<ProductResponse> getProductsList (@RequestParam(required = false) Long brandId){
```

Comment:
```suggestion
    public List<ProductResponse> getProductsList (Long brandId){
```
참고로, 이 경우 `@RequestParam(required = false)`를 생략하더라도 똑같이 동작합니다.